Sources/FontFeatures.swift

```diff
@@ -97,6 +97,34 @@
 
     }
 
+    /// A feature provider for displaying a fraction.
+    public enum Fractions: FontFeatureProvider {
+
+        /// No fraction formatting
+        case none
+
+        /// Diagonal Fractions, when written on paper, are written on one line
+        /// with the numerator diagonally above and to the left of the
+        /// demoninator, with the slash ("/") between them.
+        case diagonal
+
+        /// Vertical Fractions, when written on paper, are written on one line
+        /// with the numerator directly above the
+        /// demoninator, with the slash ("/") lying horizontally between them.
```

Comment:
#nth this may be misleading, because at this point it's no longer a slash, but a fully horizontal line. This is just a guess, because I can't find any fonts that support this feature in practice.